### DISCUSSION TITLE
is it possible to work on issue without running node ?

### DISCUSSION THREAD

**Discussion by BrazilRaw** [paradigmxyz/reth#4257] [Help]
i really want to work on reth issue but i don't have node, is it possible to do it guys?

**Comment by Rjected** [comment]
> i really want to work on reth issue but i don't have node, is it possible to do it guys?

yes! It depends on what the impact of the issue is, please comment on an issue if you want to work on it! It's also a great idea to join the telegram group in the README if you haven't already.

Along with running the existing unit / integration tests, sometimes running a sepolia node is all that's needed to test new contributions

**Reply by BrazilRaw** [reply]
Hello thank you for your fast answer. I was thinking working on this issue https://github.com/paradigmxyz/reth/issues/4255 i have never really worked on ethereum issue this could be my first issue. But without node i'm fearing if i fail on it

**Comment by mattsse** [comment]
see my comment on #4255
